.github/libs/versionUpdater.js

```diff
@@ -24,12 +24,15 @@ const getVersionNumberFromString = (versionString) => {
  * @param {Number} major
  * @param {Number} minor
  * @param {Number} patch
- * @param {Number} [build]
+ * @param {Number} build
```

Comment:
I think this is still an optional parameter, so it should have a default and square brackets in the JSDoc to mark it as optional:

```javascript
/**
 * ...
 * @param {Number} [build]
 * @returns {String}
 */
const getVersionStringFromNumber = (major, minor, patch, build = 0) => {
    return `${major}.${minor}.${patch}-${build}`;
}
```